.github/workflows/main.yml

```diff
@@ -11,6 +11,27 @@ on:
   workflow_dispatch:
 
 jobs:
+  shellcheck:
+    name: Shellcheck
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Run ShellCheck for host scripts (POSIX)
+        uses: ludeeus/action-shellcheck@master
```

Comment:
We would prefer to use a fixed version of this container (1.1.0 appears to be the most recent)